fury/shaders/marker_billboard_dec.frag

```diff
@@ -0,0 +1,100 @@
+in float vMarker;
+uniform float markerOpacity;
+uniform float edgeOpacity;
+uniform float edgeWidth;
+uniform vec3 edgeColor;
+uniform mat4 MCDCMatrix;
+uniform mat4 MCVCMatrix;
+
+float ndot(vec2 a, vec2 b ) {
+    return a.x*b.x - a.y*b.y;
+};
```

Comment:
Some compilers are more picky with the syntax than others. On GLSL you can not have ; at the end of a scope. For some reason. Replace all }; by } in the shaders.
This will fix the bug @skoudoro experienced

Reply:
Done in 0ece7bb

Reply:
perfect, works well